tgui/packages/tgui/interfaces/EFTPOS.js

```diff
@@ -21,94 +36,82 @@ export const EFTPOS = (props, context) => {
 const LockedView = (props, context) => {
   const { act, data } = useBackend(context);
   const {
-    transaction_purpose,
     transaction_amount,
-    linked_account,
     transaction_paid,
   } = data;
   return (
-    <Section title="Current Transaction" mt={1}>
-      <LabeledList>
-        <LabeledList.Item label="Transaction Purpose">
-          {transaction_purpose}
-        </LabeledList.Item>
-        <LabeledList.Item label="Value">
-          {/* Ternary required otherwise the 0 is offset weirdly */}
-          {transaction_amount ? transaction_amount : '0'}
-        </LabeledList.Item>
-        <LabeledList.Item label="Linked Account">
-          {linked_account ? linked_account : 'None'}
-        </LabeledList.Item>
-        <LabeledList.Item label="Actions">
-          <Button
-            content={transaction_paid ? 'Reset' : 'Reset (Auth required)'}
-            icon="unlock"
-            onClick={() => act('toggle_lock')}
-          />
-        </LabeledList.Item>
-      </LabeledList>
-      <NoticeBox mt={1}>
-        <Button
-          content="------"
-          icon="id-card"
-          mr={2}
-          onClick={() => act('scan_card')}
-        />
+    <>
+      <Box mt={2} bold width="100%" fontSize="3rem" color={transaction_paid ? 'green' : 'red'} align="center" justify="center">Payment Due: ${transaction_amount}</Box>
```

Comment:
For `Payment Due` perhaps it can be a ternary check with "Payment Accepted" if `transaction_paid` is true, or "Payment Due" if not.

Just makes things clearer.

Reply:
added